homeassistant/components/tractive/__init__.py

```diff
@@ -226,9 +251,13 @@ async def _listen(self) -> None:
                     if server_was_unavailable:
                         _LOGGER.debug("Tractive is back online")
                         server_was_unavailable = False
+                    # Support both deprecated wellness_overview and new health_overview
```

Comment:
Tractive has deprecated the `wellness_overview` message (the API no longer sends these messages), we should remove all code related to `wellness_overview`.

Reply:
Agreed, i didn't want to 'overstep' :D